docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,10 @@ develop
     *    - Type
          - Change
 
+    *    - |userbreak|
+         - The previously deprecated RocksDBKVS has been removed.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1966>`__)
+
```

Comment:
going on on the above, do we want to say that AtlasDB Service Server by default no longer persists its state?